spec/spec_helper.rb

```diff
@@ -4,7 +4,7 @@
 SimpleCov.start
 
 require File.expand_path("#{File.dirname(__FILE__)}/../config/boot")
-require 'byebug'
```

Comment:
weird to have this, but no reference in gem_spec

Reply:
it probably would have worked if you happened to have that gem already installed, but yes, best bet is to have in the gemspec